### DISCUSSION TITLE
Getting error "Document must be JSON object, got string" when trying view "Live Documentation"

### DISCUSSION THREAD

**Discussion by kiranpatel8** [ApicurioArchive/apicurio-studio#2334] [Q&A]
I made a change to one of the "paths" by adding two additional query parameters. Then whe I try to view it, I get the following error
**Error: Document must be JSON object, got string**
    at t.detectOpenAPI (https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:66643)
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:29459
    at Generator.next (<anonymous>)
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:28947
    at new Promise (<anonymous>)
    at r (https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:28692)
    at x (https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:29265)
    at https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:33317
    at Generator.next (<anonymous>)
    at a (https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js:8:28749)

ReDoc Version: 2.0.0-rc.75
Commit: d7440fb

I removed the two changes and still keep getting the error. I am not able "save" or even make a clone of the api. 

Is there a way to go back to a previous "one day old" version of the api or a way to fix the issue? What could be causing this error?
Please help me, I have a project deadline and I have created this api toi share it with my vendors and they are waiting. Now it seems like I cannot view the Live Documerntation since I am getting the above error.

**Comment by EricWittmann** [comment]
Super late on this answer, and apologies for the trouble you had using Studio.  One lesson for anyone who sees this discussion in the future is that the Try-Live version of Studio is not a production system.  We have no SLAs or anything like that, and data integrity is not guaranteed.  Despite this our track record is pretty good.

But in this case something unknown went wrong and I didn't catch this discussion post.  :(

Likely what happened is a bug in one of the operations implementing the changes you made, which will in turn cause all sorts of downstream havoc as you've seen.  I realize it's too late now - I do hope there was a backup copy somewhere, or you were able to copy/paste the source from an existing editor view.
